sysconfigs/daint/config.yaml

```diff
@@ -6,7 +6,10 @@ config:
     tcl:
   build_stage:
   extensions:
-  install_path_scheme: '{name}/{version}/{compiler.name}/{hash}'
+  install_tree:
+    projections:
+      all: '{name}/{version}/{compiler.name}/{hash}'
```

Comment:
would there be a way to also add a subfolder corresponding to the compiler version in the path? Maybe there are good reasons not to do it.

Reply:
yes of course, it's just {compiler.version}. I don't remember exactly how this was decided but it was a long time ago (two years ago). Is this urgent or would it be fine for you if we talk about it in the incoming spack-meeting? I will add an issue in any case

Reply:
#360 :)

Reply:
Thanks Elsa. No not urgent at all, just brain storming on the PR where you get your hands dirty ;-)